proto/connectrpc/conformance/v2/service.proto

```diff
@@ -243,3 +262,75 @@ message Header {
   string name = 1;
   repeated string value = 2;
 }
+
+// RawHTTPRequest models a raw HTTP request. This can be used to craft
+// custom requests with odd properties (including certain kinds of
+// malformed requests) to test edge cases in servers.
+message RawHTTPRequest {
+  string verb = 1;
```

Comment:
Could we model this with an enum?

Reply:
We could. I don't know that it's useful, and that would prevent a test case where we send a weird verb in the request.

Reply:
Gotcha, no worries. Just a thought.